File: car-listing-service/src/main/java/io/classiccrew/car_listing_service/entity/CarRentalTerms.java
package io.classiccrew.car_listing_service.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class CarRentalTerms extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO, generator = "native")
    private Long carRentalTermId;
    private Long carId;
    private int minRentPeriod;
    private int maxRentPeriod;
    private double pricePerDay;
}

File: car-listing-service/rental/core/entities/car_rental_terms.py
from entities.base_entity import BaseEntity


class CarRentalTerms(BaseEntity):

    def __init__(self,
                 car_rental_term_id: int,
                 car_id: int,
                 min_rent_period: int,
                 max_rent_period: int,
                 price_per_day: float,
                 created_at=None, created_by=None, updated_at=None, updated_by=None):

        super().__init__(created_at, created_by, updated_at, updated_by)

        self.car_rental_term_id = car_rental_term_id
        self.car_id = car_id
        self.min_rent_period = min_rent_period
        self.max_rent_period = max_rent_period
        self.price_per_day = price_per_day
